### DISCUSSION TITLE
How do I configure enqueued jobs to be processed only on the master instance?

### DISCUSSION THREAD

**Discussion by ashramsey** [jobrunr/jobrunr#503] [Q&A] [answered]
Hi, our Spring Boot app is in Fargate and has multiple instances. I would like to know how I would configure Jobrunr to only process  enqueued jobs on one instance (master?), or even just to know if it's not possible?

A similar question was asked in 2021, but no answers were given, hoping I might get one this time round 🙏🏼 

Thanks.

**Comment by rdehuyss** [comment, marked as answer]
Hi @ashramsey - this is possible using server tags in the pro version.

All the best,
Ronald
